pkg/kubelet/cm/helpers_linux.go

```diff
@@ -187,6 +187,19 @@ func ResourceConfigForPod(pod *v1.Pod, enforceCPULimits bool, cpuPeriod uint64)
 		result.CpuShares = &shares
 	}
 	result.HugePageLimit = hugePageLimits
+
+	if enforceMemoryQoS {
+		memoryMin := int64(0)
+		if request, found := reqs[v1.ResourceMemory]; found {
```

Comment:
maybe it's in the KEP, but why do you treat only memory.request here? (I would expect a `memoryHigh` based on it)

Reply:
Yes, as the KEP proposed, "memory.min specifies a minimum amount of memory the cgroup must always retain, i.e., memory that can never be reclaimed by the system. If the cgroup's memory usage reaches this low limit and can’t be increased, the system OOM killer will be invoked. We map it to requests.memory"